src/queue.js

```diff
@@ -99,11 +99,11 @@ export const sendMultipartMessage = async (config, room, response, msg) => {
 
     if (numParts > maxMessageParts) {
         await msg.reply(`I wrote a poem of ${numParts} messages for you!`);
-        await wait(2);
```

Comment:
The reason why I set this to 1 & 2 seconds, is that we already pre-muted 2 seconds for each part (I just realised I should have used `numParts` instead), so we want to get the parts out as fast as possible within this mute.

Reply:
I just realized too I did not notice it was length of response and not the number of parts referenced in the `lastActivityTime` :(  Pushed.

Reply:
I don't see where we pre-muted, though, sorry, can you clarify? `lastActivityTime` does not affect the multipart message itself. The issue here is that we can only send 2 messages in short succession, then we land into chat's native throttle. Maybe the delay between part 1 and 2 could be 1 second and 2 for the rest?